mirdata/datasets/indexes/slakh_index_baby.json

```diff
@@ -0,0 +1,3717 @@
+{
+  "version": "baby",
+  "tracks": {
+    "Track00001-S00": {
+      "audio": [
+        "babyslakh_16k/Track00001/stems/S00.wav",
+        "ea0e7b3d996bb3fedfbf9ee43b5c414f"
+      ],
+      "midi": [
+        "babyslakh_16k/Track00001/MIDI/S00.mid",
+        "68f9d227a4fd70acdcd80a5bd3b69e22"
+      ],
+      "metadata": [
+        "babyslakh_16k/Track00001/metadata.yaml",
+        "ffde21b0625fd72ba04103ca55f6765d"
+      ]
+    },
+    "Track00001-S01": {
+      "audio": [
+        "babyslakh_16k/Track00001/stems/S01.wav",
+        "9e1ff049ce7dbb270efcde28c9392206"
+      ],
+      "midi": [
+        "babyslakh_16k/Track00001/MIDI/S01.mid",
+        "4f7d02354ff2ff7711bfc239e51ac8ee"
+      ],
+      "metadata": [
+        "babyslakh_16k/Track00001/metadata.yaml",
+        "ffde21b0625fd72ba04103ca55f6765d"
+      ]
+    },
+    "Track00001-S02": {
+      "audio": [
+        "babyslakh_16k/Track00001/stems/S02.wav",
+        "d984056672df94e1d046197c41456fe9"
+      ],
+      "midi": [
+        "babyslakh_16k/Track00001/MIDI/S02.mid",
+        "ebc174f4f8a1c89c2a8c9ef8b9d9e3ae"
+      ],
+      "metadata": [
+        "babyslakh_16k/Track00001/metadata.yaml",
+        "ffde21b0625fd72ba04103ca55f6765d"
+      ]
+    },
+    "Track00001-S03": {
+      "audio": [
+        "babyslakh_16k/Track00001/stems/S03.wav",
+        "98002d3f0cb3ce9ff59660efb8cc6f3f"
+      ],
+      "midi": [
+        "babyslakh_16k/Track00001/MIDI/S03.mid",
+        "4bf9331befcaa0dbb02322b9426130dd"
+      ],
+      "metadata": [
+        "babyslakh_16k/Track00001/metadata.yaml",
+        "ffde21b0625fd72ba04103ca55f6765d"
+      ]
+    },
+    "Track00001-S04": {
+      "audio": [
+        "babyslakh_16k/Track00001/stems/S04.wav",
+        "8b190fb8f3774a4891cf6eeec899f2cb"
+      ],
+      "midi": [
+        "babyslakh_16k/Track00001/MIDI/S04.mid",
+        "31c56abf0d23cf683f09eb69bbe0b9ea"
+      ],
+      "metadata": [
+        "babyslakh_16k/Track00001/metadata.yaml",
+        "ffde21b0625fd72ba04103ca55f6765d"
+      ]
+    },
+    "Track00001-S05": {
+      "audio": [
+        "babyslakh_16k/Track00001/stems/S05.wav",
+        "ba3991d7b6942c6394dbec55b968d63c"
+      ],
+      "midi": [
+        "babyslakh_16k/Track00001/MIDI/S05.mid",
+        "c40df0d092b1eb74d4c6279ef141c6be"
+      ],
+      "metadata": [
+        "babyslakh_16k/Track00001/metadata.yaml",
+        "ffde21b0625fd72ba04103ca55f6765d"
+      ]
+    },
+    "Track00001-S06": {
```

Comment:
what's the deal with these effects tracks with null audio and midi? why do they have to be tracks and not metadata? could the record object solve this?

Reply:
Yeah, this was one of my doubts I'm waiting for ethan on. I'm not sure if they're included in the mixture or not - if they're not i'm inclined to remove them as tracks?

Reply:
yes. I think it's better to remove them from the index or from the multi-track list. I am curious what do we do with effects. there are some room impulse responses in phenicx anechoic which were used to build the multi-microphone recordings but I left them out.